### PR TITLE
Chore: Disable SASS deprecation warnings #DS-1514

### PR DESCRIPTION
**Pull request by curdaj**

### CODE REVIEW

File: apps/demo/config/vite/app.ts [hunk 1]
```diff
@@ -65,6 +65,7 @@ export default defineConfig({
           join(pathRelativeToRepositoryRoot, 'node_modules'),
           join(pathRelativeToRepositoryRoot, 'node_modules/@lmc-eu/spirit-design-tokens/src/scss'),
         ],
+        silenceDeprecations: ['mixed-decls'],
```
Comment (literat) [inline apps/demo/config/vite/app.ts:68]: **suggestion (if-minor):** Can you please add a comment to this like `// @see https://sass-lang.com/documentation/breaking-changes/mixed-decls/` so we have the reference what does it solves?

Maybe also mention that we are silencing false-positives...?

Comment (crishpeen) [inline apps/demo/config/vite/app.ts:68]: maybe we should add a comment why we do this 😬

Comment (crishpeen) [reply apps/demo/config/vite/app.ts:68]: :D see what @literat said

Comment (curdaj) [reply apps/demo/config/vite/app.ts:71]: added

Comment (curdaj) [reply apps/demo/config/vite/app.ts:71]: added

**Review by crishpeen — APPROVED** [review]

**Pull request merged**